STEER/STEERBase/AliOADBContainer.cxx

```diff
@@ -225,22 +220,28 @@ void  AliOADBContainer::CleanDefaultList()
   fDefaultList->Delete();
 }
 
+void  AliOADBContainer::CleanLists()
+{
+  // Clean lists
+  if (fArray)       fArray->Delete();
+  if (fDefaultList) fDefaultList->Delete();
+  if (fPassNames)   fPassNames->Delete();
+}
+
 Int_t AliOADBContainer::GetIndexForRun(Int_t run, TString passName) const
 {
   //
   // Find the index for a given run 
   
   Int_t found = 0;
   Int_t index = -1;
-  for (Int_t i = 0; i < fEntries; i++) 
-    {
-	if (fPassNames) if (fPassNames->At(i)) if (passName.CompareTo(fPassNames->At(i)->GetName())) continue;
-	if (run >= fLowerLimits[i] && run <= fUpperLimits[i])
-	{
-	  found++;
-	  index = i;
-	}
+  for (Int_t i = 0; i < fEntries; i++) {
+    if (fPassNames) if (fPassNames->At(i)) if (passName.CompareTo(fPassNames->At(i)->GetName())) continue;
```

Comment:
This line will be interpreted by modern compilers as "Misleading indentation" - should be three separate lines.

Reply:
Ok that probably was in the code before. I usually expanded such statements, when I saw them.